### PR TITLE
V2 migration

### PR DESCRIPTION
**Pull request by muhamadazmy**
Migration too to v3. it runs as a daemon until it's possible to match a farm payout address from v3 with the v2 farm it's currently on.

### CODE REVIEW

File: pkg/environment/environment.go [hunk 1]
```diff
@@ -58,13 +60,15 @@ var (
 	envDev = Environment{
 		RunningMode: RunningDev,
 		BcdbURL:     "https://explorer.devnet.grid.tf/explorer",
+		GraphQlURL:  "https://graphql.dev,grid.tf/graphql",
```
Comment (LeeSmet) [inline pkg/environment/environment.go]:
```suggestion
		GraphQlURL:  "https://graphql.dev.grid.tf/graphql",
```

**Review by LeeSmet — CHANGES_REQUESTED** [review]

File: cmds/modules/migrate/main.go [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+	if err := crossFingers(); err != nil {
+		return errors.Wrap(err, "failed to reboot")
+	}
+	log.Info().Msg("I should never show up on screen")
+	return nil
+}
+
+func crossFingers() error {
+	f, err := os.OpenFile("/proc/sysrq-trigger", os.O_WRONLY, 0400)
```
Comment (maxux) [inline cmds/modules/migrate/main.go]:
```go
syscall.Sync()
err = syscall.Reboot(syscall.LINUX_REBOOT_CMD_RESTART)
```

File: pkg/environment/environment.go (hunk 1 above)
Comment (muhamadazmy) [reply pkg/environment/environment.go]: Oh good catch, luckily we have no devnet anyway

File: cmds/modules/migrate/main.go (hunk 2 above)
Comment (muhamadazmy) [reply cmds/modules/migrate/main.go]: Yup, I started this with a "script" mindset.

**Pull request merged**